src/main/java/mate/service/DriverService.java

```diff
@@ -0,0 +1,18 @@
+package mate.service;

```

Comment:
```suggestion
package mate.jdbc.service;
```
please put everything in mate.jdbc package